CMakeLists.txt

```diff
@@ -11,6 +11,8 @@ SET (CMAKE_VERBOSE_MAKEFILE true)
 #SET (ESCARGOT_OUTPUT "shell" CACHE STRING "ESCARGOT_OUTPUT")
 SET (ESCARGOT_ASAN "0" CACHE STRING "ESCARGOT_ASAN")
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-mismatched-new-delete")
```

Comment:
Why did you insert this flag?

Reply:
I'm using gcc 11.2.0 and for me if i'm not using this flag the build will fail. I forgot to delete it.